### ISSUE TITLE
Akka HTTP client with akka.actor.typed.ActorSystem not working

### ISSUE THREAD

**Issue by politrons** [akka/akka-http#3528]
Hi all,

I was following the documentation, to use Akka HTTP client with Akka typed, but none of your code examples compile

https://doc.akka.io/docs/akka-http/current/introduction.html#streaming

Since in the example Akka HTTP() it require an implicit of `akka.actor.ActorSystem` but the implicit you're defining is from `akka.actor.typed.ActorSystem`

Any example of how to use Akka HTTP client with akka.actor.typed.ActorSystem ? 

Regards

**Comment by jrudolph** [comment]
Thanks for the report, @politrons. Which versions of Akka and Akka HTTP do you use? It should work with Akka HTTP 10.2.x and Akka 2.6.10.

You can also try creating an initial project with from our quickstart at https://github.com/akka/akka-http-quickstart-scala.g8.

**Comment by sgc109** [comment]
It doesn't work for me either. Versions I'm using are 2.6.8 for Akka and 10.2.1 for Akka HTTP. I just copied [this code](https://doc.akka.io/docs/akka-http/current/introduction.html#using-akka-http) on the documentation. It seems that the package path is not `akka.actor.typed.ActorSystem`, but `akka.actor.ActorSystem`. But, it doesn't work yet because there is not property `executionContext` in ActorSystem.

**Comment by raboof** [comment]
> I just copied [this code](https://doc.akka.io/docs/akka-http/current/introduction.html#using-akka-http) on the documentation.

Can you be a bit more specific? Which file+line is the error on, and what is the exact error message?

**Comment by sgc109** [comment]
@raboof 
Oh I found the reason. On the doc, there is no dependency specified for akka actor ("com.typesafe.akka" %% "akka-actor" % AkkaVersion,). So I just added `"com.typesafe.akka" %% "akka-actor" % AkkaVersion,` as a parameter in Seq.apply() method. But, it turns out, you guys are using `akka-actor-typed` not `akka-actor`. @politrons might be doing the same as me. Let me do a PR for adding dependency of akka-actor-typed if it is ok.
